docs/vsphere-csi.md

```diff
@@ -55,7 +55,7 @@ spec:
   resources:
     requests:
       storage: 1Gi
-  storageClassName: Space-Efficient
+  storageClassName: mongodb-sc
```

Comment:
I cannot get the point of this change.

`Space-Efficient` is shown as `STORAGECLASS` at line 90.
In addition, I cannot find the storageclass name mongodb-sc in this doc.

Reply:
You're correct. I forgot to also change the prompt.
I was referring to the [`mongodb-storageclass.yaml`](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md#create-a-storageclass) Example linked as requirement [at line 44](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/vsphere-csi.md?plain=1#L44). Following this guide, the `StorageClass` has the name `mongodb-sc` and the VM storage policy `Space-Efficient`.
If my assumptions are correct, I can also change the prompt to make the doc consistent.

Reply:
I see, thanks for your explanation.
`Space-Efficient` is not a good name as you specified, we need to update the prompt also to avoid confusion.

Reply:
Alright, updated the prompt with the proper StorageClass name 👍